app/code/core/Mage/Api2/Model/Resource.php

```diff
@@ -201,7 +201,7 @@ public function dispatch()
             case self::ACTION_TYPE_ENTITY . self::OPERATION_CREATE:
                 // Creation of objects is possible only when working with collection
                 $this->_critical(self::RESOURCE_METHOD_NOT_IMPLEMENTED);
-                break;
+                // no break
```

Comment:
Even though `$this->_critical()` throws an exception, the comment `// no break` is confusing, it implies the process continues. I prefer not to touch this. What do you think?

Reply:
I can remove this [rule ](https://cs.symfony.com/doc/rules/control_structure/no_break_comment.html) or change the comment text. What would you prefer?